Actuator.m

```diff
@@ -198,8 +198,32 @@ function tune_pid_controller(obj, saturation, p_gain, i_gain, dv_gain, de_gain)
             configuration = [p_gain, i_gain, dv_gain, de_gain , saturationL_H];
             obj.write_multi_registers(133, 6, configuration);  % PC_PGAIN 133, num consecutive registers 6, register values  configuration
         end
+
         
+        %% Change Motor Direction
+        % Sets the motor direction to 0 or 1
+        function set_direction(obj, direction)
+            obj.write_register(152, direction);   %POS_SIGN
+        end
+
+        %% Configure AutoZero
+        % Set autozero with configuration:
+        %ZERO_MODE: 0: zero negative positions, 1: manual zeroing, 2: Auto Zero
+        %Enabled, 3:Auto Zero on Boot
+        %EXIT_MODE: 1: sleep, 2: force, 3: position,4: Haptic,5:Kinematic
+        function configure_zero(obj, ZERO_MODE, AUTO_ZERO_FORCE_N, AUTO_ZERO_EXIT_MODE)
+            configuration = [ZERO_MODE AUTO_ZERO_FORCE_N AUTO_ZERO_EXIT_MODE];
```

Comment:
In the newest update, there is a speed register that is not contiguous with the others. However, this obviously will cause issues if used on an older firmware version.

AUTO_ZERO_SPEED_MMPS          177

Reply:
Good to know, is the default reasonable? If so, everything should still work with this code no?